packages/esm-ohri-pmtct-app/src/views/maternal-health/tabs/antenatal-care.component.tsx

```diff
@@ -31,10 +31,10 @@ const AntenatalCareList: React.FC<AntenatalCareListProps> = ({ patientUuid }) =>
         },
       },
       {
-        key: 'hivTestResults',
-        header: t('hivTestResults', 'HIV Test Results'),
+        key: 'hivTestStatus',
+        header: t('hivTestStatus', 'HIV Test Status'),
```

Comment:
@Herobiam Is this part of the translations? You could just quickly add and you'll be good to go